### PR TITLE
ci: adjust approve

### PR DESCRIPTION
**Pull request by zombieJ**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      -     |
| 🇨🇳 Chinese |     -      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by yoyo837 — DISMISSED** [review]

File: .github/workflows/visual-regression-diff-approver.yml [hunk 1]
```diff
@@ -25,13 +25,27 @@ jobs:
       id: check_approval
       uses: actions/github-script@v7
       with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
         script: |
+          const issue_number = context.payload.issue.number;
```
Comment (crazyair) [inline .github/workflows/visual-regression-diff-approver.yml:30]: 为啥不写个 js，然后在这里调用呢？

File: .github/workflows/visual-regression-diff-approver.yml [hunk 2]
```diff
@@ -25,13 +25,27 @@ jobs:
...
+          console.log('PR:', issue_number);
+
+          const prResponse = await github.rest.pulls.get({
```
Comment (crazyair) [inline .github/workflows/visual-regression-diff-approver.yml:33]: async 没有，可以直接 await 吗？

Comment (kiner-tang) [reply .github/workflows/visual-regression-diff-approver.yml:33]: 有没有可能在 github ci 中的 script 默认是被包裹在一个顶层 async 匿名函数下面的呢

Comment (kiner-tang) [reply .github/workflows/visual-regression-diff-approver.yml:33]: https://github.com/marketplace/actions/github-script#run-a-separate-file-with-an-async-function

**Review by MadCcc — APPROVED** [review]

**Review by crazyair — APPROVED** [review]

**Pull request merged**